.github/workflows/e2e-testing.yml

```
@@ -13,12 +14,28 @@
         with:
           node-version: ${{ vars.NODE_VERSION }}
 
+   
+      - name: Install Dependencies
+        run: npm install
+
+  
+      - name: Start server
+        run: npm run start & 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)